exporters/otlp/src/otlp_http_client.cc

```diff
@@ -21,14 +21,15 @@
 #include "google/protobuf/stubs/common.h"
 
 #if defined(GOOGLE_PROTOBUF_VERSION) && GOOGLE_PROTOBUF_VERSION >= 3007000
+#  include "google/protobuf/stubs/stringpiece.h"
```

Comment:
Wondering why it is necessary to include stringpiece.h because `Base64Escape` is declared in `strutil.h`?

Reply:
yeah thanks, removed it.